### PR TITLE
F/520 responsive design

### LINKED ISSUE #520: handle 4 databases on comparison screen
**Is your feature request related to a problem? Please describe:**  
with my screen size, the comparison-screen styling and formatting breaks with 4 compared databases.

**Please describe the solution you'd like:**  
We could develop a responsive design, which uses the tile-width to style the tile-title etc. (do not use just the number of dbs, because 4 databases look good on 4k-screens)

**Please describe alternatives you've considered:**  
we could also limit the max comparable databases to 4.

**Affected Component(s):**  
Frontend

### PR DESCRIPTION
**Pull request by MaxSchneider1337**
# Resolves #520

**Does your pull request solve a problem? Please describe:**  
adds responsiveness to comparison screen.

**Affected Component(s):**  
Frontend

### CODE REVIEW

File: hyrisecockpit/frontend/src/components/container/MetricTile.vue [hunk 1]
```diff
@@ -49,8 +52,10 @@ import {
 import { Database } from "../../types/database";
 import DatabaseChip from "@/components/details/DatabaseChip.vue";
 import TimeInterval from "@/components/details/TimeInterval.vue";
+import { eventBus } from "@/plugins/eventBus";
 
 interface Props extends MetricProps {
+  isThin: boolean;
```
Comment (schTi) [inline hyrisecockpit/frontend/src/components/container/MetricTile.vue:58]: I dont know, but isThin doesnt seem to be good naming here. Maybe you could find a more functional name here

File: hyrisecockpit/frontend/src/components/container/MetricTile.vue [hunk 2]
```diff
@@ -49,8 +52,10 @@ import {
 import { Database } from "../../types/database";
 import DatabaseChip from "@/components/details/DatabaseChip.vue";
 import TimeInterval from "@/components/details/TimeInterval.vue";
+import { eventBus } from "@/plugins/eventBus";
```
Comment (schTi) [inline hyrisecockpit/frontend/src/components/container/MetricTile.vue:55]: this import isnt used anywhere

File: hyrisecockpit/frontend/src/components/container/MetricsComparisonTable.vue [hunk 3]
```diff
@@ -36,6 +37,7 @@ import { useDatabaseFlex } from "../../meta/components";
 import MetricTile from "@/components/container/MetricTile.vue";
 
 interface Data {
+  needsThinTiles: Readonly<Ref<boolean>>;
```
Comment (schTi) [inline hyrisecockpit/frontend/src/components/container/MetricsComparisonTable.vue:40]: also naming here

**Review by schTi — CHANGES_REQUESTED** [review]

**Comment by MaxSchneider1337** [comment]
closed due to old dev version

**Pull request closed without merging**